changelogs/unreleased/gh-8931-box-info-election-leader-name.md

```diff
@@ -0,0 +1,3 @@
+## feature/box
+
+* Added `leader_name` field to the box.info.election (gh-8931).
```

Comment:
```suggestion
* Added the `leader_name` field to `box.info.election` (gh-8931).
```

Reply:
@Astronomax, please apply the doc team's suggestion.